playground_server_design.md

```diff
@@ -0,0 +1,130 @@
+# SQLFlow Playground Server
+
+SQLFlow Playground Server expose an REST API service which enable users to share
+the resources in one playground cluster. Users can take advantage of SQLFlow by
+installing a small plugin on her/his own Jupyter Notebook.
+
+## The Architecture
+
+**SQLFlow Playground Server** is a side-by-side service of our playground cluster.
```

Comment:
What is a "side-by-side service"?

Reply:
By using `side-by-side`, I mean this service should be started along the playground. Without the playground, this service is not meaningful.

Reply:
That should be a `side-car service`?